src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Tree/Dependencies/DependenciesGraphProviderTests.cs

```diff
@@ -626,14 +626,14 @@ public class DependenciesGraphProviderTests
             await provider.BeginGetGraphDataAsync(mockGraphContext);
 
             // Assert
-            Assert.Equal(5, outputNodes.Count);
+            Assert.Equal(4, outputNodes.Count);
 
             var topNode1Result = GetNodeById(outputNodes, topNode1.Id);
             Assert.True(topNode1Result.HasCategory(CodeNodeCategories.ProjectItem));
             Assert.Equal(1, topNode1Result.OutgoingLinkCount);
 
             var topNode2Result = GetNodeById(outputNodes, topNode2.Id);
-            Assert.Equal(2, topNode2Result.OutgoingLinkCount);
+            Assert.Equal(1, topNode2Result.OutgoingLinkCount);
```

Comment:
Note: this change conflicts with a similar change in https://github.com/dotnet/roslyn-project-system/pull/1584